client/src/interface/User.ts

```diff
@@ -1,6 +1,10 @@
 export interface User {
-  email: string;
   username: string;
+  email: string;
+  strategy: string;
+  id: number;
+  picture: string;
+  mongoId: number;
```

Comment:
Could you explain this here? MongoIDs aren't numbers and there is already an id which is also a number too?